src/utils/tests/patches.test.js

```diff
@@ -436,3 +456,45 @@ describe('utils.js tests', () => {
     comparePatch(patch[0], '/spec', { running: false });
   });
 });
+
+describe('getDeviceBootOrderPatch() tests', () => {
+  const disks = [
+    { disk: {}, name: 'disk-one', bootOrder: 1 },
+    { disk: {}, name: 'disk-two', bootOrder: 2 },
+    { disk: {}, name: 'disk-three', bootOrder: 3 },
```

Comment:
can you please add the non bootable cloud init here as well?

Reply:
Done!